front-end/stateofthecards-front-end/src/App.tsx

```diff
@@ -0,0 +1,27 @@
+import React from "react";
+import { BrowserRouter as Router, Switch, Route } from "react-router-dom";
+import Login from "./Login";
+import Dashboard from "./Dashboard";
+import styles from "./App.module.css";
+
+const App = () => (
+	<Router>
+		<div className="App">
+			<div className={styles.contentPanel}>
+				<Switch>
+					<Route path="/dashboard">
+						<Dashboard />
+					</Route>
+					<Route path="/">
+						<Login />
+					</Route>
+				</Switch>
+			</div>
+			<div className={styles.alertPanel}>
+				<h1>Please turn the device horizontally. (Landscape Mode)</h1>
```

Comment:
"horizontally" -> "on its side"
Would be more user friendly I think

Reply:
If someone doesn't know the word horizontally, he or she should not be placing card games online in my opinion...